terraform/deployments/github/import.tf

```
@@ -1,4 +1,4 @@
 import {
-  to = github_repository.govuk_repos["govuk-content-publishing-guidance"]
-  id = "govuk-content-publishing-guidance"
-}
+  to = github_branch_protection.govuk_repos["select-with-search-component"]
```

We first need to import existing GitHub repository and then add GitHub branch protection for it.

```suggestion
  to = github_repository.govuk_repos["select-with-search-component"]
```